crates/generators/python/item.py

```diff
@@ -21,11 +21,40 @@
     else:
         durabilities[variant] = f"Some({durability})"
 
-output_data = generate_enum("Item", items)
+output_data = "use serde::{Serialize, Deserialize};"
+
+output_data += generate_enum("Item", items, derives=["Serialize", "Deserialize"],
+                                                     prelude="#[serde(try_from = \"String\", into = \"&'static str\")]")
 output_data += generate_enum_property("Item", "id", "u32", ids, True)
 output_data += generate_enum_property("Item", "name", "&str", names, True, "&'static str")
 output_data += generate_enum_property("Item", "display_name", "&str", display_names, True, "&'static str")
 output_data += generate_enum_property("Item", "stack_size", "u32", stack_sizes)
 output_data += generate_enum_property("Item", "durability", "Option<u32>", durabilities)
 
+output_data += f"""
+    use std::convert::TryFrom;
+
+    impl TryFrom<String> for Item {{
```

Comment:
This should probably be `FromStr` instead of `TryFrom<String>`. Or we can implement both traits.

Reply:
We implement `TryFrom<String>` in order to use `#[serde(try_from = String)]` but we can also implement `FromStr` as we have also implemented `Into<&'static str>`.